src/reducers/books.js

```diff
@@ -0,0 +1,17 @@
+const initialState = [{ title: 'React', category: 'software' }];
```

Comment:
- [ ] [OPTIONAL] You could have this initial state inside a initialState.js file to organize your code better.